cypress/e2e/http_block.ts

```diff
@@ -3,7 +3,100 @@ import { loadFlowCode } from '../support/helper';
 // tslint:disable: quotemark
 /// <reference types="Cypress" />
 
-describe('HTTP Block Integration Tests', () => {
+
```

Comment:
Unfortunately changing the file name made code review a bit more difficult.
To help here some info: 
- I divided the test in two test suits: a normal http request and paginated request
- The new code is inside the first test suit: `HTTP Block Request`